go_api/docs/docs.go

```diff
@@ -285,6 +285,31 @@ const docTemplate = `{
                 }
             }
         },
+        "/event-users":{
+            "post":{
+                tags: ["event-users"],
+                "description":"eventとuserの中間テーブルへのInsert",
+                "parameters": [
+                    {
+                        "name": "event_id",
+                        "type": "integer",
+                        "in": "query",
+                        "description": "EventID",
+                    },
+                    {
+                        "name": "user_id",
+                        "type": "integer",
+                        "in": "query",
+                        "description": "UserID",
+                    }
+                ],
+                "responses":{
+                    "200":{
+                        "description":"Created Event-User",
```

Comment:
```suggestion
                        "description":"Created Event-Users",
```
ここだけ一応sつけた方がいいと思います。

Reply:
確かにそうすね！
修正してmergeします